src/http.ps1

```diff
@@ -35,8 +35,11 @@ function HttpSend {
 	}
 	$ch.UseProxy = $false
 	$cli = [Net.Http.HttpClient]::new($ch)
-	$resp = $cli.SendAsync($Req, [Net.Http.HttpCompletionOption]::ResponseHeadersRead).GetAwaiter().GetResult()
-	return $resp
+	try {
+		return $cli.SendAsync($Req, [Net.Http.HttpCompletionOption]::ResponseHeadersRead).GetAwaiter().GetResult()
+	} catch {
+		throw "An error occured while initiating an HTTP request; check your network and try again.`n+ $_"
```

Comment:
This works fine. However, I think I would prefer if the cause of the exception was more-clearly separated from the error message. Something like this:
```suggestion
		throw "An error occured while initiating an HTTP request; check your network connection and try again.`n+ Caused by: $_"
```

Another option would be to nest the exception. But, I realize PowerShell is terrible at printing out error messages with stack traces and caused by / inner exception information:
```suggestion
		throw New-Object System.Exception("An error occured while initiating an HTTP request; check your network connection and try again.", $_.Exception)
```